travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
```

Comment:
Rename: `.travis.yaml`

Reply:
Ah! but add the leading `.`!

Reply:
sorry. I messed that up in my first comment

Reply:
ah I missed that part. Fixed :)